docs/skills/memory.md

```diff
@@ -32,8 +36,21 @@ class RememberSkill(Skill):
     async def remember(self, message):
         information = await self.opsdroid.memory.get("remember")
         await message.respond(information)
+        
+    @match_regex(r'forget it')
+    async def remember(self, message):
+        await self.opsdroid.memory.delete("remember")
+        await message.respond("Ok I'll forget it")
 ```
 
-In the above example we have defined two skill functions. The first takes whatever the user says after the word "remember" and stores it in the database.
+In the above example we have defined three skill functions. The first takes whatever the user says after the word "remember" and stores it in the database.
 
 The second retrieves and prints out that text when the user says "remind me".
+
+The third deletes what is remembered in the database when the user says "forget it".
+## Reference
+
+```eval_rst
+.. autoclass:: opsdroid.memory.Memory
+    :members:
+    ```
```

Comment:
Sorry to be a pain, you will need to move these backticks to the beginning of the line yourself, I couldn't do it when making the suggestion

Reply:
```eval_rst
.. autoclass:: opsdroid.memory.Memory
    :members:
```
Like this?

Reply:
Yeah haha